Contest_API.md

```diff
@@ -427,19 +449,17 @@ Countdown is resumed by setting a new `start_time` and resetting
 
 #### PATCH start\_time
 
-To replace the *Contest Start Interface*, at the ICPC World
-Finals, an API provided by a CCS or CDS implementing this specification
-must have a role that has the ability to clear or set the contest start
-time via a PATCH method.
+Implementations must have an admin role that has the ability to clear or set the
+contest start time via a PATCH method.
```

Comment:
I prefer "a role that..." over "an admin role that..."

Reply:
This is a WF specific requirement, so I think we should leave this optional and then require it in the CCS requirements document.

Reply:
Hmmmm but the pausing of contest time is generally useful (and has been used outside of WF for sure). I’m not sure what part is WF specific?

Reply:
Well, it's useful, but it's not critical for functioning. So I don't think this should be a requirement in general.

Reply:
I think this is one that is clearly useful as an optional feature which we can do when we have the role/feature idea?

Reply:
New commit removes the 'admin'.

This was required before and I wasn't trying to change that via this PR. I totally agree we should implement optional features and this should be one of them (and required for finals).

Reply:
> pausing of contest time is generally useful

I would argue that pausing of contest time is a general anti-feature... but that's another discussion :).

> This was required before and I wasn't trying to change that via this PR. I totally agree we should implement optional features and this should be one of them (and required for finals).

I agree that this is outside the score of this PR. I also agree with @eldering and @deboer-tim that it should be moved to teh CCS requirements document.